### PR TITLE
Proposed changes to lifecycle to clarify incubation, graduation and exit.

### PR DESCRIPTION
**Pull request by dheckenberg**
Signed-off-by: Daniel Heckenberg <daniel.heckenberg+github@gmail.com>

### CODE REVIEW

File: process/lifecycle.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This document provides for three lifecycle stages for contributed projects:
 * Adopted; and
 * Archived
 
-All projects must meet the Incubation stage requirements and following a successful Graduation Review, progress to Adopted. It is possible that some projects may be approved as Incubation and pass a Graduation Review at the same time to advance directly to Adopted. A project that does not pass a Graduation Review remains at the Incubation stage unless the TAC votes to transition the project to Archive status.
+All projects must meet the Incubation stage requirements and following a successful Graduation Review, progress to Adopted. It is possible that some projects may be approved as Incubation and pass a Graduation Review at the same time to advance directly to Adopted. A project that does not pass a Graduation Review remains at the Incubation stage unless the TAC votes to transition the project to Archive status.  Alternatively, the TAC or the project TSC may vote to exit the lifecycle under specific terms for transfer of trademarks and other key assets.
```
Comment (dkg-animallogic) [inline process/lifecycle.md]: What's intended by the "specific terms" clause for exiting the lifecycle?

File: process/lifecycle.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ To be accepted at the Incubation stage, a project must:
 * Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC, or the TAC’s designated recipient for contribution proposals.
 * Provide such additional information as the TAC may reasonably request.
 * Be available to present to the TAC with respect to the project’s proposal and inclusion in ASWF. Project teams should be prepared to present a detailed (20-30 minutes in length) overview on the project in addition to speaking to the information contained in the project contribution proposal.
-* Be deemed by the TAC to add value to the mission of ASWF.
+* Be deemed by the TAC to add potential or value to the mission of ASWF.
```
Comment (dkg-animallogic) [inline process/lifecycle.md]: Should this be "potential value or value" instead of "potential or value"

File: process/lifecycle.md (hunk 1 above)
Comment (dheckenberg) [reply process/lifecycle.md]: I was attempting to capture the fact that specifics of transfer arrangements and so on would need to be part of any exit vote.  cf Brexit.

File: process/lifecycle.md (hunk 2 above)
Comment (dheckenberg) [reply process/lifecycle.md]: potential value is an improvement -- thanks.

File: process/lifecycle.md (hunk 1 above)
Comment (jmertic) [reply process/lifecycle.md]: Looks good to me.

**Pull request merged**